tests/test_tacacs_server_group.rb

```diff
@@ -231,11 +241,14 @@ def test_remove_server_twice_tacacs
     assert_empty(servers, 'Error: Collection not empty')
 
     detach_aaaservergroup(aaa_group)
-    detach_tacacsserverhost(server1)
-    detach_tacacsserverhost(server2)
+    detach_tacacsserverhost(server_name1)
+    detach_tacacsserverhost(server_name2)
   end
 
   def test_get_vrf_tacacs
+    # Not part of NetDev development
```

Comment:
These are added because the netdev type does not support these properties correct?  If so, I would prefer one of the following options (in order of preference).

1) Add an `exclude: [ios_xr]` for these properties in the yaml and then use the `validate_property_excluded` to do the `assert_nil` and assert_raises(Cisco::UnsupportedError)` checks
  - https://github.com/cisco/cisco-network-node-utils/blob/56982bca060bcab5a13dcb2124dce4a02ebe54d9/tests/test_vlan_private.rb#L76
2) Change one is simple, but based on our agreement we will accept the PR without this change, but please add a TODO reminder in the comments to validate this and we will make the updates.